content/blog/2024/04/2024-04-12-election-announcement.md

```diff
@@ -0,0 +1,102 @@
++++
+date = "2024-04-12"
```

Comment:
```suggestion
date = "2024-04-12T14:00:00Z"
```

Specifying the time looks better on the blog. Zola falls back to 00:00 which looks odd